frontend/app/src/utils/balances.ts

```diff
@@ -69,17 +69,19 @@ export const mergeAssetBalances = (
 export const groupAssetBreakdown = (
   breakdowns: AssetBreakdown[]
 ): AssetBreakdown[] => {
-  const grouped = groupBy(breakdowns, 'location');
+  const grouped = groupBy(
+    breakdowns.map(breakdown => ({
+      ...breakdown,
+      key: breakdown.location + breakdown.address
+    })),
+    'key'
+  );
```

Comment:
this will mean double loop... could be achieved in one loop using `breakdowns.reduce()`